tsfc/fiatinterface.py

```diff
@@ -178,6 +173,15 @@ def _(element, vector_is_mixed):
         # Real element is just DG0
         cell = element.cell()
         return create_element(ufl.FiniteElement("DG", cell, 0), vector_is_mixed)
+    if element.family() == "Quadrature":
+        # Sneaky import from FFC
+        from ffc.quadratureelement import QuadratureElement
+        ffc_element = QuadratureElement(element)
+
+        def tabulate(self, order, points, entity):
+            return QuadratureElement.tabulate(self, order, points)
+        ffc_element.tabulate = types.MethodType(tabulate, ffc_element)
+        return ffc_element
```

Comment:
FFC elements are "FIAT elements" not-implementing the full FIAT element interface. But this might not be an obstacle for your suggestion. I agree that it is quite a chaos now. But I'm not sure, and you'll surely know more, how the suggestion fits an eventual FInAT transition and FIAT retirement.